applications/RANSApplication/python_scripts/formulations/utilities.py

```diff
@@ -179,21 +194,25 @@ def InitializeYPlusVariablesInConditions(model_part):
 def InitializePeriodicConditions(
     base_model_part,
     model_part,
-    variables_list):
+    variables_list,
+    periodic_condition_name = "PeriodicCondition"):
+
     properties = model_part.CreateNewProperties(
         model_part.NumberOfProperties() + 1)
-    pcu = KratosCFD.PeriodicConditionUtilities(
-        model_part, model_part.ProcessInfo[Kratos.DOMAIN_SIZE])
-    for variable in variables_list:
-        pcu.AddPeriodicVariable(properties, variable)
+
+    if (variables_list is not None):
```

Comment:
Considering that it is passed by argument, in which situation can the `variables_list` be `None`?

Reply:
I see your point, it cannot be unless developer makes a mistake. So I removed it :)